resources/function_help/json/url_encode

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "url_encode",
+  "type": "function",
+  "groups": ["Maps"],
+  "description": "Returns an URL encoded string from a map.<br>The underlying function uses <a href='https://doc.qt.io/qt-5/qurlquery.html#toString'>QUrlQuery::toString</a> with <a href='https://doc.qt.io/qt-5/qurl.html#ComponentFormattingOption-enum'>QUrl::FullyEncoded</a> flags.<br>Note that the plus sign '+' is not converted.",
```

Comment:
Is the qt stuff relevant? This link is likely to break at some stage, so I'd rather see the content which is relevant for users included directly here. (Also it's not nice to direct end users to c++ docs, that's a big turn off for non-developers)

Reply:
done